### ISSUE TITLE
CRw: Hierarchy / Back to Link / Current selection is missing

### ISSUE THREAD

**Issue by bit-pro-iew-eui** [swiss/styleguide#302]
![image](https://cloud.githubusercontent.com/assets/7510842/5219930/4c9ab1ac-765e-11e4-8d50-6d8168422585.png)

The hierarchy of the menu is not correct. "Drinks" is a level higher.
==> Make a nested list

![image](https://cloud.githubusercontent.com/assets/7510842/5392367/820ee3cc-8125-11e4-8250-4599b149b40b.png)
The "Back to..." Symbol is not read to screen readers.
==> Use ARIA "labelledby" or a hidden text.

http://swiss.github.io/styleguide/navigation_modules_-_hierarchical_navigation.html#d-page-nav-list
The current selection "Wine" isn't read by screenreaders.
==> Use ARIA "selected=true" or a hidden text.
